src/cache/QueryResultCacheOptions.ts

```diff
@@ -9,6 +9,11 @@ export interface QueryResultCacheOptions {
      */
     identifier: string;
 
+    /**
+     * Prefix to be applied to the cache identifer.
+     */
+    prefix?: string;
```

Comment:
this option should work if database cache (instead of redis) is used as well.

Reply:
Ha of course, no problems.